cli/buildengine.ts

```diff
@@ -387,14 +387,29 @@ function runDockerAsync(args: string[]) {
         if (process.platform == "darwin")
             mountArg += ":delegated"
 
+        let fullArgs = ["--rm", "-v", mountArg, "-w", "/src"].concat(dargs).concat([cs.dockerImage]).concat(args);
+        if (flags) {
+            fullArgs = flags.concat(fullArgs)
+        }
+
         return nodeutil.spawnAsync({
             cmd: "docker",
-            args: ["run", "--rm", "-v", mountArg, "-w", "/src"].concat(dargs).concat([cs.dockerImage]).concat(args),
+            args: ["run"].concat(fullArgs),
```

Comment:
Why wouldn't we just put this in `fullArgs`?

Reply:
because of the prepending that happens with the flags parameter. run always needs to come first

Reply:
Ah, makes sense.